src/main/java/frc/robot/maps/FrancoisMap.java

```diff
@@ -80,14 +89,36 @@ public SendableSpeedController spinner() {
     @Override
     public LiftMap getLiftMap() {
         return new LiftMap() {
+            CANSparkMax follower = new CANSparkMax(28, MotorType.kBrushless);
```

Comment:
This could be defined in `elevator()`. Only `leader` needed to be moved here.

Reply:
It gives complaints about variables getting closed if they don't do this